### DISCUSSION TITLE
help compiling cargo

### DISCUSSION THREAD

**Discussion by SwinyPyxel** [revoxhere/duino-coin#1640] [Q&A]
I tried putting https://github.com/revoxhere/duino-coin/wiki/How-to-compile-fasthash-accelerations and I'm stuck at the command ""cargo build --release"" and it gives me says there is no cargo, do you know why? For information I am French.

**Comment by identd113** [comment]
If you have a pi with 512MB ram, you may have some issues. Post back the exact model you have.
